index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Homepage</title>
```

Comment:
better to have his as Prime 8 | Home

Reply:
Thankyou for pointing that. Home page still work in progress. Created a basic one so that site-builder can find index.html and not face any build conflicts.

Reply:
Oh, Cool. Then you can work on it later